kowalski/alert_watcher_ztf.py

```diff
@@ -136,6 +136,36 @@ def make_photometry(alert: dict, jd_start: float = None):
     mask_good_diffmaglim = df_light_curve["diffmaglim"] > 0
     df_light_curve = df_light_curve.loc[mask_good_diffmaglim]
 
+    # convert from mag to flux
+
+    # step 1: calculate the coefficient that determines whether the
+    # flux should be negative or positive
+    coeff = df_light_curve['isdiffpos'].apply(lambda x: 1 if x in [True, 1, 'y', 'Y'] else -1)
```

Comment:
Yes, Python will handle int*float as we expect but it reads easier that the multiplier is meant to be 1.0 or -1.0
```suggestion
    coeff = df_light_curve['isdiffpos'].apply(lambda x: 1.0 if x in [True, 1, 'y', 'Y'] else -1.0)
```